src/index.html

```diff
@@ -5,8 +5,439 @@
+      <div class="card container__card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27'' Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <p class="card__product-name">
+           APPLE A1419 iMac 27" Retina <br>
+           5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p
+          class="card__product-code"
+        >
+           Product code: 195434
+        </p>
+        <div class="card__stars-wrapper">
+          <div class="stars stars--4 card__stars-block">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p
+            class="card__stars-review"
+          >
+             Reviews: 5
+          </p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-syntax">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          class="card__purchase-button "
+          href="#"
+        >
+         BUY
+        </a>
+      </div>
+      <div class="card container__card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27'' Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <p class="card__product-name">
+           APPLE A1419 iMac 27" Retina <br>
+           5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p
+          class="card__product-code"
+        >
+           Product code: 195434
+        </p>
+        <div class="card__stars-wrapper">
+          <div class="stars stars--4 card__stars-block">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p
+            class="card__stars-review"
+          >
+             Reviews: 5
+          </p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-syntax">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          class="card__purchase-button "
+          href="#"
+        >
+         BUY
+        </a>
+      </div>
+      <div class="card container__card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27'' Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <p class="card__product-name">
+           APPLE A1419 iMac 27" Retina <br>
+           5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p
+          class="card__product-code"
+        >
+           Product code: 195434
+        </p>
+        <div class="card__stars-wrapper">
+          <div class="stars stars--4 card__stars-block">
+            <div class="stars__star"></div>
```

Comment:
pls blank lines between children elements
```suggestion
            <div class="stars__star"></div>
            
```